lib/matplotlib/backends/registry.pyi

```diff
@@ -0,0 +1,15 @@
+from enum import Enum
+
+
+class BackendFilter(Enum):
+    INTERACTIVE: int
+    NON_INTERACTIVE: int
+
+
+class BackendRegistry:
+    def __init__(self) -> None: ...
```

Comment:
There's no `__init__` in the implementation; do we need  it in the typing?

Reply:
No we don't, I've removed it.